marketpulse/base/tests/test_mozillians.py

```diff
@@ -0,0 +1,135 @@
+import json
+
+from unittest import TestCase
+from urlparse import parse_qs, urlparse
+
+import responses
+
+from nose.tools import eq_, raises
+
+from marketpulse.base.mozillians import (BadStatusCode, MozilliansClient, ResourceDoesNotExist,
+                                         MultipleResourcesReturned)
+
+
+class TestMozilliansClientGet(TestCase):
+    def setUp(self):
+        self.client = MozilliansClient('http://example.com/api/v2/', 'EXAMPLE_API_KEY')
+
+    @responses.activate
+    def test_get_valid_status(self):
+        """Test that valid response is returned for 200 status code."""
+        url = 'http://example.com/api/v2/'
+        body = '{"key1": "value1", "key2": "value2"}'
+        responses.add(responses.GET, url=url, body=body, status=200,
+                      content_type='application/json')
+
+        response = self.client.get(url)
+        eq_(len(responses.calls), 1)
+        eq_(responses.calls[0].request.url, url)
+        eq_(responses.calls[0].request.headers['X-API-KEY'], 'EXAMPLE_API_KEY')
+        eq_(response, json.loads(body))
+
+    @responses.activate
+    @raises(BadStatusCode)
+    def test_get_invalid_status(self):
+        """Test that error is raised response for bad status code."""
+        url = 'http://example.com/api/v2/'
```

Comment:
Same as above. You can move this outside of the test class and just append what you need for each test (multiple occurrences)